packages/structured-clone/index.ts

```diff
@@ -1,8 +1,8 @@
-// @ts-nocheck
-
 // globalThis polyfill from https://mathiasbynens.be/notes/globalthis
 (function () {
-    if (typeof globalThis === "object") return;
+    if (typeof globalThis === "object") {
```

Comment:
Some day `structuredClone` will have wide support and I can finally drop this nonsense :-/

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/API/structuredClone, it is supported in nodejs \ge 17.0.0, but not in nodejs workers. I guess this nodejs workers are the most concerning part?

Reply:
I'm also concerned about browser support :-)

Reply:
Other browsers look fine but Safari support seems problematic. I just tried running tests with `delete globalThis.structuredClone` before every tests and it passed all (except for those failing before), maybe having this kind of automated testing would be good.